Chatify/chat/models.py

```diff
@@ -6,3 +6,31 @@
 class User(AbstractUser):
     profile_photo = models.ImageField(upload_to="profile_photo/")
     mobile_number = PhoneNumberField(unique=True)
+
+    def __str__(self):
+        return self.username
+
+
+class Group(models.Model):
+    name = models.CharField(max_length=30)
+    created = models.DateTimeField(auto_now=True)
+    user = models.ManyToManyField(User, blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Chat(models.Model):
+    message = models.CharField(max_length=5000)
+    created = models.DateTimeField(auto_now=True)
+    group = models.ForeignKey(Group, on_delete=models.CASCADE)
+    sender = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="sent_messages"
+    )
+    receiver = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="received_messages"
+    )
+    is_read = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f"{self.sender.username} to {self.receiver.username}"
```

Comment:
add new line

Reply:
![image](https://user-images.githubusercontent.com/73389336/235660993-c676ddcc-597b-4b10-b608-a07cee1a7805.png)

I have used black with the help of this it will automatically leave one line at the end of the file

Reply:
ok.